src/style.css

```
@@ -0,0 +1,139 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  padding-top: 32px;
+  padding-left: 19px;
+  padding-right: 19px;
+}
```

```suggestion
.card__image {
  width: 160px;
  height: 134px;
}
```
lets set padding to the `.card`